0032-peering-clearing-settlement/0032-peering-clearing-settlement.md

```diff
@@ -0,0 +1,160 @@
+
+However, there is an edge case where the peer's balances may be out of sync demonstrated in this example.
+
+> **Example : Alice and Bob**
+
+> Assuming they start with a balance of 0 and the following sequence of events occurs:
+
+> 1. Bob passes an ILP Prepare to Alice with an amount of 20.
+> 1. Alice replies with a valid ILP Fulfill and believes she has done so before the request expires, however Bob considers the request to have expired and has already rejected the payment to his downstream peer.
+
+> The balance changes will be as follows:
+
+> - After Step 2:
+>   - Alice's balance is 20 satoshis
+>   - Bob's balance is 0 satoshis
+
+> Clearly, Bob and Alice need to reconcile this situation. The specific mechanism they use and recourse they take to resolve this difference is outside the scope of the Interledger protocol. 
+
+It is important to note that this is an extremely unlikely scenario as telescoping expiry timeouts between peers should reduce the risk of this to almost zero.
+
+Further, this is a single packet, and a key principal of the Interledger network is to break payments into very small packets so that in the unlikely event that this does occur the losses incurred by the two parties are negligible.
+
+Nonetheless, it is likely that future bilateral protocols (like BTP) will define more detailed processes for reconciliation and clearing.
+
+# Settlement
+
+It is unlikely that the flow of packets between two peers will always net out such that the balance continuously tends toward zero. In most cases flows will be predominantly in one direction and there will be a need for the peers to reduce them back to 0.
+
+This avoids a situation where one peer owes the other a large sum of money and is then unable or unwilling to pay (settlement risk). Connectors should define a maximum Interledger account balance to manage the settlement risk they expose themselves to with their peers.
+
+Further, the principal of the Interledger network is to settle as often as is practically possible and financially viable such that this risk is very low.
+
+In the case of peers that use payment channels to settle this can be done for every packet as the cost of settlement is almost zero (the cost of exchanging claims on the channel). This reduces the settlement risk between peers down to the value of only those packets that are not yet fulfilled, rejected or expired (i.e. packets that are in-flight).
+
+Connectors will configure their own business rules regarding when to settle, based on how much they trust their peers and the costs and speed of settlement on the underlying network.
+
+For most implementations, this configuration will consist of, at a minimum, a maximum balance, and a settlement threshold. 
+
+## Maximum Balance
```

Comment:
You should explain why there's the maximum balance. If you post-pay each packet, then the peer could theoretically steal that much money from you by not paying up. If it's pre-paid, the maximum balance would probably be the total that the peer has pre-paid

Reply:
That's a good point. I may add a section on different settlement models.

Reply:
You could also include in that how you might set it up with a slower settlement method.
One really important point to get across is how the choice of settlement method, frequency, and credit limits is completely independent of what everyone else in the network does and how you are insulated from other people's choices.